classes/functions.php

```diff
@@ -58,7 +58,7 @@ public function issueLookup($comic_id) {
 			die ( "Connection failed: " );
 		}
 
-		$sql = "SELECT comics.wiki_id, comics.series_id, comics.comic_id, series.series_name, series.series_vol, comics.issue_number, comics.release_date, comics.story_name, comics.cover_image, comics.plot, comics.original_purchase FROM comics LEFT JOIN series ON comics.series_id=series.series_id WHERE comics.comic_id = $comic_id";
+		$sql = "SELECT * FROM comics LEFT JOIN series ON comics.series_id=series.series_id WHERE comics.comic_id = $comic_id";
```

Comment:
I don't think it matters much in a small application, but I did wonder if it is more efficient processing wise to only grab the columns you need rather than everything. Granted this query grabs ALMOST everything, so I don't have a problem using a wildcard.